README.md

```diff
@@ -47,31 +56,90 @@ and ask!
 
 ## When to Create a New Enhancement
 
+Enhancements should be related to work to be implemented in the near
+future. If you have an idea, but aren't planning to implement it right
+away, the conversation should start somewhere else like the mailing
+list or Slack.
+
 Create an enhancement here once you:
 
 - have circulated your idea to see if there is interest
 - (optionally) have done a prototype in your own fork
 - have identified people who agree to work on and maintain the enhancement
   - many enhancements will take several releases to complete
 
-## Why are Enhancements Tracked
-
-As the project evolves, its important that the OKD community understands how we
-build, test, and document our work.  Individually it is hard to understand how
-all parts of the system interact, but as a community we can lean on each other
-to build the right design and approach before getting too deep into an
-implementation.
-
-## When to Comment on an Enhancement Issue
-
-Please comment on the enhancement issue to:
-- request a review or clarification on the process
-- update status of the enhancement effort
-- link to relevant issues in other repos
-
-Please do not comment on the enhancement issue to:
-- discuss a detail of the design, code or docs. Use a linked-to-issue or design PR
-  for that
+## How are Enhancements Reviewed and Approved?
+
+The author of an enhancement is responsible for managing it through
+the review and approval process, including soliciting feedback on the pull request
+and in meetings, if necessary.
+
+Each enhancement should have at least one "approver" and several
+reviewers designated in the header of the document.
+
+The approver assists authors who may not be familiar with the process,
+the project, or the maintainers. They may provide advice about who
+should review a specific proposal and point out deadlines or other
+time-based criteria for completing work. The approver is responsible
+for recognizing when consensus has been reached so that a proposal is
```

Comment:
It sounds like then that the approval will come after the reviews are completed. Could we be explicit about that for this role?

Reply:
```suggestion
for recognizing when consensus among reviewers has been reached so that a proposal is
```